NoClassDefFoundError: com/google/android/collect/Maps

When I run tests, it throw NoClassDefFoundError 

```
java.lang.NoClassDefFoundError: com/google/android/collect/Maps
    at org.robolectric.shadows.ShadowMediaMetadataRetriever.<clinit>(ShadowMediaMetadataRetriever.java:21)
    at org.robolectric.RobolectricBase.reset(RobolectricBase.java:1811)
    at org.robolectric.Robolectric.reset(Robolectric.java:1372)
    at org.robolectric.internal.ParallelUniverse.resetStaticState(ParallelUniverse.java:37)
    at org.robolectric.RobolectricTestRunner$2.evaluate(RobolectricTestRunner.java:207)
    at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
    at org.robolectric.RobolectricTestRunner$1.evaluate(RobolectricTestRunner.java:158)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
    at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:86)
    at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:49)
    at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:69)
    at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:48)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
    at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
    at org.gradle.messaging.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
    at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
    at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
    at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:105)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
    at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
    at org.gradle.messaging.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:360)
    at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: com.google.android.collect.Maps
    at org.robolectric.bytecode.AsmInstrumentingClassLoader.getByteCode(AsmInstrumentingClassLoader.java:172)
    at org.robolectric.bytecode.AsmInstrumentingClassLoader.findClass(AsmInstrumentingClassLoader.java:126)
    at org.robolectric.bytecode.AsmInstrumentingClassLoader.loadClass(AsmInstrumentingClassLoader.java:98)
    ... 40 more
```

My androidstudio is fine but my friend one throw this error. (gradle clean, build works fine)

--- a/robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowMediaMetadataRetriever.java
+++ b/robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowMediaMetadataRetriever.java
@@ -4,26 +4,22 @@ import android.content.Context;
 import android.graphics.Bitmap;
 import android.media.MediaMetadataRetriever;
 import android.net.Uri;
-
-import com.google.android.collect.Maps;
-
+import java.io.FileDescriptor;
+import java.util.HashMap;
+import java.util.Map;
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
 import org.robolectric.annotation.Resetter;
 import org.robolectric.shadows.util.DataSource;
-
-import java.io.FileDescriptor;
-import java.util.HashMap;
-import java.util.Map;
 
 import static org.robolectric.shadows.util.DataSource.toDataSource;
 
 @Implements(MediaMetadataRetriever.class)
 public class ShadowMediaMetadataRetriever {
   private DataSource dataSource;
-  private static final Map<DataSource, HashMap<Integer, String>> metadata = Maps.newHashMap();
-  private static final Map<DataSource, HashMap<Long, Bitmap>> frames = Maps.newHashMap();
-  private static final Map<DataSource, RuntimeException> exceptions = Maps.newHashMap();
+  private static final Map<DataSource, Map<Integer, String>> metadata = new HashMap<>();
+  private static final Map<DataSource, Map<Long, Bitmap>> frames = new HashMap<>();
+  private static final Map<DataSource, RuntimeException> exceptions = new HashMap<>();
 
   public void setDataSource(DataSource dataSource) {
     RuntimeException e = exceptions.get(dataSource);
@@ -33,7 +29,7 @@ public class ShadowMediaMetadataRetriever {
     }
     this.dataSource = dataSource;
   }
-  
+
   @Implementation
   public void setDataSource(String path) {
     setDataSource(toDataSource(path));
@@ -71,9 +67,9 @@ public class ShadowMediaMetadataRetriever {
   /**
    * Configures an exception to be thrown when {@link #setDataSource}
    * is called for the given data source.
-   * 
+   *
    * @param ds the data source that will trigger an exception
-   * @param e the exception to trigger, or <tt>null</tt> to 
+   * @param e the exception to trigger, or <tt>null</tt> to
    * avoid throwing an exception.
    */
   public static void addException(DataSource ds, RuntimeException e) {
@@ -95,7 +91,7 @@ public class ShadowMediaMetadataRetriever {
    * you can call {@link #addMetadata(DataSource, int, String)} directly.
    *
    * @param path the path to the data source whose metadata is being set.
-   * @param keyCode the keyCode for the metadata being set, as used by {@link MediaMetadataRetriever#extractMetadata(int)}. 
+   * @param keyCode the keyCode for the metadata being set, as used by {@link MediaMetadataRetriever#extractMetadata(int)}.
    * @param value the value for the specified metadata.
    */
   public static void addMetadata(String path, int keyCode, String value) {
@@ -108,14 +104,14 @@ public class ShadowMediaMetadataRetriever {
     }
     frames.get(ds).put(time, bitmap);
   }
-  
+
   /**
    * Adds the given bitmap at the given time for the given data source.
    * Uses <tt>path</tt> to call {@link org.robolectric.shadows.util.DataSource#toDataSource(String)} and
    * then calls {@link #addFrame(DataSource, long, Bitmap)}. This
    * method is retained mostly for backwards compatibility;
    * you can call {@link #addFrame(DataSource, long, Bitmap)} directly.
-   * 
+   *
    * @param path the path to the data source.
    * @param time the playback time at which the specified bitmap
    * should be retrieved.

--- a/robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityManagerTest.java
+++ b/robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityManagerTest.java
@@ -1,19 +1,19 @@
 package org.robolectric.shadows;
 
-import org.junit.Test;
-import org.junit.Before;
-import org.junit.runner.RunWith;
-import org.robolectric.Robolectric;
-import org.robolectric.RuntimeEnvironment;
-import org.robolectric.TestRunners;
-import static org.assertj.core.api.Assertions.*;
-
-import java.util.List;
-import com.google.android.collect.Lists;
+import android.accessibilityservice.AccessibilityServiceInfo;
 import android.content.pm.ServiceInfo;
 import android.view.accessibility.AccessibilityManager;
-import android.accessibilityservice.AccessibilityServiceInfo;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.TestRunners;
+
 import static android.content.Context.ACCESSIBILITY_SERVICE;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.robolectric.Shadows.shadowOf;
 
 @RunWith(TestRunners.WithDefaults.class)
@@ -42,21 +42,21 @@ public class ShadowAccessibilityManagerTest {
 
   @Test
   public void shouldReturnExpectedEnabledServiceList() {
-    List<AccessibilityServiceInfo> expected = Lists.newArrayList(new AccessibilityServiceInfo());
+    List<AccessibilityServiceInfo> expected = new ArrayList<>(Arrays.asList(new AccessibilityServiceInfo()));
     shadowAccessibilityManager.setEnabledAccessibilityServiceList(expected);
     assertThat(accessibilityManager.getEnabledAccessibilityServiceList(0)).isEqualTo(expected);
   }
 
   @Test
   public void shouldReturnExpectedInstalledServiceList() {
-    List<AccessibilityServiceInfo> expected = Lists.newArrayList(new AccessibilityServiceInfo());
+    List<AccessibilityServiceInfo> expected = new ArrayList<>(Arrays.asList(new AccessibilityServiceInfo()));
     shadowAccessibilityManager.setInstalledAccessibilityServiceList(expected);
     assertThat(accessibilityManager.getInstalledAccessibilityServiceList()).isEqualTo(expected);
   }
 
   @Test
   public void shouldReturnExpectedAccessibilityServiceList() {
-    List<ServiceInfo> expected = Lists.newArrayList(new ServiceInfo());
+    List<ServiceInfo> expected = new ArrayList<>(Arrays.asList(new ServiceInfo()));
     shadowAccessibilityManager.setAccessibilityServiceList(expected);
     assertThat(accessibilityManager.getAccessibilityServiceList()).isEqualTo(expected);
   }

--- a/robolectric/src/test/java/org/robolectric/shadows/ShadowAutoCompleteTextViewTest.java
+++ b/robolectric/src/test/java/org/robolectric/shadows/ShadowAutoCompleteTextViewTest.java
@@ -5,13 +5,13 @@ import android.widget.ArrayAdapter;
 import android.widget.AutoCompleteTextView;
 import android.widget.Filter;
 import android.widget.Filterable;
-import com.google.android.collect.Lists;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.RuntimeEnvironment;
 import org.robolectric.TestRunners;
-
-import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
@@ -46,7 +46,7 @@ public class ShadowAutoCompleteTextViewTest {
       FilterResults results = new FilterResults();
       if (text != null) {
         results.count = 2;
-        results.values = Lists.newArrayList("Foo", "Bar");
+        results.values = new ArrayList<>(Arrays.asList("Foo", "Bar"));
       }
       return results;
     }

